tests/qos/test_tunnel_qos_remap.py

```diff
@@ -346,7 +346,7 @@ def pfc_pause_test(storm_handler, peer_info, prio, ptfadapter, dut, port, queue,
         start_pfc_storm(storm_handler, peer_info, prio)
         ptfadapter.dataplane.flush()
         # Record the queue counter before sending test packet
-        base_queue_count = get_queue_counter(dut, port, queue, True)
+        base_queue_count = get_queue_counter(dut, port, queue, False)
```

Comment:
Why do we need to modify the clear_before_read value here? Have we encountered the issue caused by this? thanks.

Reply:
test logic is comparing queue counter valure, ensure no increasing for queue counter. not necessary to clear counter to get zero base value.